packages/fxa-admin-panel/src/components/AppLayout/index.scss

```diff
@@ -30,3 +30,24 @@
 main {
   flex: 4;
 }
+
+// custom styles from when Header lived in this package; it has since been
+// pulled into `fxa-shared` and the leftover styles were placed here.
+.header-page {
+  background: left 15% / cover no-repeat url('../../images/ff-branding-bg.svg')
+    var(--color-primary);
+  padding: var(--app-padding-x-min);
+
+  h1 {
+    align-self: center;
+    color: var(--color-white);
+    font-size: var(--header-font-size-lg);
+    font-weight: var(--font-weight-semi-bold);
+    margin: 0 0 0 20px;
+    text-shadow: 1px 1px 3px var(--color-black);
+  }
+
+  .logo {
+    width: 50px;
+  }
```

Comment:
If I'm reading this right, most of these can go in the tsx?

Reply:
Just to circle back, I added a more descriptive comment above. We need to get Tailwind setup in the admin panel and styles showing up in `fxa-react` Storybook, likely Jody or I will work on that next sprint.